.golangci.yml

```diff
@@ -9,3 +9,27 @@ linters:
     - lll
     - wsl
     - gomnd
+    - depguard
```

Comment:
Why exctacly these linters below were disabled?

Reply:
Enable some linters and fix errors. I disabled only linter "testpackage".